nautobot/docs/plugins/development.md

```diff
@@ -785,7 +785,7 @@ For more advanced usage, you may want to instead inherit from one of Nautobot's
 | [Tags](../models/extras/tag.md) | ❌ | ❌ | ❌ | ✅ |
 
 !!! note
-    When using `OrganizationalModel` or `PrimaryModel`, you also must use the `@extras_features` decorator to specify support for (at a minimum) the `"custom_fields"` and `"relationships"` features.
+    When using `OrganizationalModel` or `PrimaryModel`, you also can use the `@extras_features` decorator to specify support for features e.g `"webhooks""`, `"graphql"`.
```

Comment:
I'd vote for just removing this note, it's no longer critical. Also the edit here is misleading as you can use `@extras_features` on `BaseModel` subclasses too...